druid-shell/src/backend/mac/window.rs

```diff
@@ -1198,7 +1214,21 @@ impl WindowHandle {
     pub fn show_titlebar(&self, _show_titlebar: bool) {}
 
     // Need to translate mac y coords, as they start from bottom left
-    pub fn set_position(&self, position: Point) {
+    pub fn set_position(&self, mut position: Point) {
+        // TODO: Maybe collin can get this into a state where modal windows follow the parent?
+        // There is an API to do child windows, (https://developer.apple.com/documentation/appkit/nswindow/1419152-addchildwindow)
```

Comment:
I'm not sure you can move the parent while a modal is presented?

Reply:
Tooltips and dropdowns suffer from the same issue, we expect them to be closed when moving a window but ATM that is upto the user to program. It might still be worth implementing for those? if not Ill remove the comment.

Reply:
> ATM that is upto the user to program.

is it possible? We don't have any windowmove events

Reply:
uhh oh. If I am going to be implementing proper parent tracking, I can only do this for GTK and with some research probably X11. Others will have to help me on different platforms.

Reply:
I think we should give windowmove events and then the user should do whatever they like (move with the window or close the popup)

Reply:
Ok, lets put that in another MR to not bloat this one.